### PR TITLE
Add settings to navigation drawer

### LINKED ISSUE #575: Move Settings to Drawer
So they are reachable from everywhere

### PR DESCRIPTION
**Pull request by anselm92**
Fixes #575
screenshot_20171111-142248

### CODE REVIEW

File: app/src/main/java/de/tum/in/tumcampusapp/auxiliary/DrawerMenuHelper.java [hunk 1]
```diff
@@ -94,6 +99,12 @@ public void populateMenu(Menu navigationMenu) {
                              .setIntent(new Intent(mContext, item.activity));
             }
         }
+        SubMenu appMenu = navigationMenu.addSubMenu("App");
```
Comment (kordianbruck) [inline app/src/main/java/de/tum/in/tumcampusapp/auxiliary/DrawerMenuHelper.java]: This needs to be translated using the Strings.xml.

Comment (pfent) [reply app/src/main/java/de/tum/in/tumcampusapp/auxiliary/DrawerMenuHelper.java]: I'd just leave the submenu unnamed...

Comment (kordianbruck) [reply app/src/main/java/de/tum/in/tumcampusapp/auxiliary/DrawerMenuHelper.java]: Just checked the google apps and others: neither use a submenu name, so we might as well leave it out then.

Comment (anselm92) [reply app/src/main/java/de/tum/in/tumcampusapp/auxiliary/DrawerMenuHelper.java]: I agree, i first tried it that way, however leaving the subname empty will create a nasty space between the divider and the settings submenu entry. 
Already googled to find a solution for that, unfortunately this seems to be possible only by inflating a menu.xml and using a group.
We could move the submenus into a xml file, inflate it and add the entries programmatically. What do you think?

Comment (kordianbruck) [reply app/src/main/java/de/tum/in/tumcampusapp/auxiliary/DrawerMenuHelper.java]: Is it not possible to directly add an entry to the menu without a submenu? 
If not, then a inflated .xml should be fine

Comment (pfent) [reply app/src/main/java/de/tum/in/tumcampusapp/auxiliary/DrawerMenuHelper.java]: I think you can just [add](https://developer.android.com/reference/android/view/Menu.html#add(int,%20int,%20int,%20java.lang.CharSequence)) the item with another groupid, which creates a divider

Comment (kordianbruck) [reply app/src/main/java/de/tum/in/tumcampusapp/auxiliary/DrawerMenuHelper.java]:
```
use NONE if an item should not be in a group.
```

**Comment by anselm92** [comment]
@pfent you were right, worked by adding it directly to the navmenu

File: app/src/main/java/de/tum/in/tumcampusapp/auxiliary/DrawerMenuHelper.java (hunk 1 above)
Comment (anselm92) [reply app/src/main/java/de/tum/in/tumcampusapp/auxiliary/DrawerMenuHelper.java]: @pfent you were right, worked by adding it directly to the navmenu

**Comment by kordianbruck** [comment]
@anselm92 did you force push here as well? No need, I squash commits when I merge them.

**Comment by anselm92** [comment]
@kordianbruck yes i did. ok i'll do that for future commits :)

**Pull request merged**